utxo/utxo_meta.go

```diff
@@ -187,3 +187,75 @@ func (uv *UtxoVM) UpdateForbiddenContract(param *pb.InvokeRequest, batch kvdb.Ba
 	uv.metaTmp.ForbiddenContract = param
 	return err
 }
+
+func (uv *UtxoVM) LoadIrreversibleBlockHeight() (int64, error) {
+	irreversibleBlockHeightBuf, findErr := uv.metaTable.Get([]byte(ledger_pkg.IrreversibleBlockHeightKey))
+	if findErr == nil {
+		utxoMeta := &pb.UtxoMeta{}
+		err := proto.Unmarshal(irreversibleBlockHeightBuf, utxoMeta)
+		return utxoMeta.GetIrreversibleBlockHeight(), err
+	}
+	return int64(0), nil
```

Comment:
it is safe to return 0 only if findErr=="not found"

Reply:
> it is safe to return 0 only if findErr=="not found"

Thank you for your suggestion and i will correct all of them.